mobile/src/components/auth/auth.tsx

```diff
@@ -44,7 +44,9 @@ const Auth: FC = () => {
       <Image source={{ uri: getImageUri(logo) }} style={styles.logo} />
       <View style={styles.circle1} />
       <View style={styles.circle2} />
-      <ScrollView>{getScreen(name)}</ScrollView>
+      <ScrollView keyboardShouldPersistTaps="handled">
```

Comment:
Create a custom scrollview component, that will wrap scrollview from react native and set keyboardShouldPersistTaps